assets/javascripts/scripts.js

```diff
@@ -0,0 +1,150 @@
+/**
+ * togglePressed() toggles the aria-pressed atribute between true or false
+ *
+ * @param ( id object) button to be operated on
+ *
+ * @return N/A
+ */
+function togglePressed(element) {
+
+  // reverse the aria-pressed state
+  if (element.attr('aria-pressed') == 'true') {
+    element.attr('aria-pressed', 'false');
+  }
+  else {
+    element.attr('aria-pressed', 'true');
+  }
+}
+
+var parseMediumJSON = function(post_rows) {
+  // max 10 posts
+  post_rows = post_rows || 10;
+
+  // get the template
+  var template_markup = $('#post-template').html();
+
+  // get the data
+  $.getJSON('https://8k431k2y7h.execute-api.us-west-2.amazonaws.com/Production/medium', function(data) {
+    var author_list = data.body.payload.references.User;
+    var all_posts = data.body.payload.posts;
+    var posts_to_output = $.map(all_posts, function(post, index){
```

Comment:
instead of using index and doing the check, we could just slice all_posts down to the appropriate length and map from there.